pages/[slug].tsx

```diff
@@ -28,20 +40,24 @@ export default function Page(props: Props): ReactElement {
         otherPosts={props.otherPosts}
       />
     );
-  } else {
-    return <RichPage page={content} />;
+  } else if (aboutPageTabs.find((tab) => tab.slug === content.slug)) {
```

Comment:
I might be wrong on this but won't it run on every rerender? Do we need to make it a function with a useMemo or useCallback?

Reply:
like `isAboutPageTab`

Reply:
I don't think this is an issue as these pages are statically generated and no longer actually change tabs but navigate to pages

Reply:
All good then. Resolved